pkg/config/validation.go

```diff
@@ -268,6 +273,26 @@ func validatePatches(patchesValidation ...patchesValidation) error {
 	return nil
 }
 
+func ValidatePlatformProject(ctx context.Context, config *config.Config, loadedConfig *cliconfig.CLI) error {
+	platformConfig, err := config.GetPlatformConfig()
+	if err != nil {
+		return fmt.Errorf("get platform config: %w", err)
```

Comment:
This will render vCluster CLI OSS not usable without a platform anymore

Reply:
I'm not sure if I understood it because I tested locally without a running platform.

Errors returned from `config.GetPlatformConfig()` are only errors related to yaml parsing.
https://github.com/loft-sh/vcluster/blob/main/config/config.go#L849-L857